Formula/kibana.rb

```diff
@@ -2,8 +2,8 @@ class Kibana < Formula
   desc "Analytics and search dashboard for Elasticsearch"
   homepage "https://www.elastic.co/products/kibana"
```

Comment:
Let's try the same as https://github.com/Homebrew/homebrew-core/pull/59860#discussion_r472946157

```suggestion
  homepage "https://www.elastic.co/kibana"
```